images/recycler/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# This is the default deployment strategy image for OpenShift Origin. It expects a set of
+# environment variables to parameterize the deploy:
+#
+#   KUBERNETES_MASTER - the address of the OpenShift master
+#   KUBERNETES_DEPLOYMENT_ID - the deployment identifier that is running this build
+#
+# The standard name for this image is openshift/origin-deployer
+#
+FROM openshift/origin
```

Comment:
They do, through deployments and builds.

On Nov 10, 2015, at 8:34 AM, Eric Paris <notifications@github.com> wrote:

In images/recycler/Dockerfile
<https://github.com/openshift/origin/pull/5792#discussion_r44429166>:

> @@ -0,0 +1,20 @@
> +#
> +# This is the default deployment strategy image for OpenShift Origin. It expects a set of
> +# environment variables to parameterize the deploy:
> +#
> +#   KUBERNETES_MASTER - the address of the OpenShift master
> +#   KUBERNETES_DEPLOYMENT_ID - the deployment identifier that is running this build
> +#
> +# The standard name for this image is openshift/origin-deployer
> +#
> +FROM openshift/origin

nodes don't have openshift containers pulled today (they might when people
have containerized node, but that's not what is done today)

RHEL7/CentOS/Fedora seems like the better base...

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5792/files#r44429166>.